images/openwisp_websocket/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # hadolint ignore=DL3007
-FROM openwisp/openwisp-base:latest
+FROM openwisp/openwisp-base:stable
```

Comment:
now that I see this here, it's misleading here, `latest` was better.

Reply:
I think, we should use "edge"/"dev" everywhere in the code with an option to override the tag when required. If someone is trying to build the images from source, they most likely would want the versioh which is present in the code.